### PR TITLE
add tests for json formatters

### PR DESCRIPTION
**Pull request by alexsavio**
This PR adds tests for the JSON formatters

### CODE REVIEW

File: lambda_handlers/formatters/tests/test_json_formatter.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+class TestInputFormatJSON:
+
+    def test_valid(self, data_sample):
+        assert data_sample == InputFormat.json(json.dumps(data_sample))
+
+    def test_empty_content(self):
+        assert {} == InputFormat.json('{}')
+
+    def test_random_string(self):
+        with pytest.raises(json.decoder.JSONDecodeError, match='Expecting value:'):
```
Comment (sleistner) [inline lambda_handlers/formatters/tests/test_json_formatter.py]: any reason to leave the match parameter truncated?

Comment (alexsavio) [reply lambda_handlers/formatters/tests/test_json_formatter.py]: Added the whole messages

File: lambda_handlers/formatters/tests/test_json_formatter.py [hunk 2]
```diff
@@ -41,6 +45,10 @@ class TestOutputFormatJSON:
     def test_valid(self, data_sample):
         assert json.dumps(data_sample) == output_format.json(data_sample)
 
+    def test_data_sample(self, data_sample):
+        content = json.dumps(data_sample)
```
Comment (sleistner) [inline lambda_handlers/formatters/tests/test_json_formatter.py]: any objections making the content more explicit as in `'{"foo": "bar"}'`?

Comment (alexsavio) [reply lambda_handlers/formatters/tests/test_json_formatter.py]: do you mean to remove the data_sample fixture and write one test for each valid type?

Comment (sleistner) [reply lambda_handlers/formatters/tests/test_json_formatter.py]: maybe something like this, less dynamic

Comment (alexsavio) [reply lambda_handlers/formatters/tests/test_json_formatter.py]: done?

**Pull request merged**